### DISCUSSION TITLE
Standard Button

### DISCUSSION THREAD

**Discussion by ysds** [twbs/bootstrap#31159] [General]
This is one of my main problem that I've been having from before join Bootstrap team.

In my experience, many applications have a white standard button:

Microsoft Fluent UI
![image](https://user-images.githubusercontent.com/4065765/85528669-2a7d2800-b647-11ea-9e90-2df31ceb5825.png)

Apple macOS
![image](https://user-images.githubusercontent.com/4065765/85529039-9e1f3500-b647-11ea-99d5-e5ad63952252.png)

GitHub Primer
![image](https://user-images.githubusercontent.com/4065765/85530517-f73b9880-b648-11ea-8e12-00a4bdbe4591.png)

So what is the best way to use like that style with Bootstrap?

* Bootstrap recommend to use the secondary button?
But, the secondary button are filled medium gray by the default. It have a different look than white or light gray with gray border used in many apps. Also the Bootstrap button variants are generated by mixin based on the theme variables, so changing the appearance of the secondary button is difficult. In addition, considering the consistency among multiple components makes its design complicated.
* Or Bootstrap recommend to use the outlined secondary or dark outlined button? (#25006)
My concern is those no semantic class names and the border colors aren't same the form control's border color.

Many applications and design systems have a white (or light gray) bacground and gray bordered secandary (standard) button. Bootstrap should have like those standard button by default? This may be like v3's `btn-default‘ revival.

However, there are many hurdles to pushing this forward.

* Should we add a new standard button variant? Or change the design of the secondary button?
* If we add a new button, what should the class name be?
* How should we handle the `.btn-secondary`?
* Should we revisit theme variants generation logic?
* Is there any impact on other components that have theme variants.
* Compatibility with v4

**Comment by patrickhlauke** [comment]
in addition, it would be nice to have a completely / mostly neutral button style by default - thinking about the proposed change here for the sidebar #31098 and having to essentially hack away at our button styles to get at what is a fairly simple design (and arguably, more useful in general than the `.btn-link` style.
